pkg/engine/task/task_apply.go

```diff
@@ -118,7 +118,7 @@ func applyResources(rr []runtime.Object, ctx Context) ([]runtime.Object, error)
 		switch {
 		case apierrors.IsNotFound(err): // create resource if it doesn't exist
 			if err := addLastAppliedConfigAnnotation(r); err != nil {
-				return nil, fmt.Errorf("failed to add last applied config annotation to %s/%s: %w", key.Namespace, key.Name, err)
+				return nil, fmt.Errorf("failed to add last applied config annotation to a %s %s: %w", r.GetObjectKind().GroupVersionKind(), key, err)
```

Comment:
how does `GroupVersionKind()` look as a string?

Reply:
https://github.com/kubernetes/apimachinery/blob/master/pkg/runtime/schema/group_version.go#L161

Reply:
@gerred you're making me look lazy! :DD thanks!